book/quickstart.nim

```diff
@@ -0,0 +1,108 @@
+import nimib, nimibook
+
+nbInit(theme = useNimibook)
+
+nbText:
+  """
+# Quickstart
+
+## Installation
+
+```shell
+$ nimble install -y nim-intops
+```
+
+Add intops to your .nimble file:
+
+```nim
+requires "nim-intops"
+```
+
+
+## Usage
+
+The most straightforward way to use intops is by importing `intops` and calling the operations from it:
+"""
+
+nbCode:
+  import intops
+
+  echo carryingAdd(12'u64, 34'u64, false)
+
```

Comment:
Is it possible to add `doAssert carryingAdd(12'u64, 34'u64, false) == (res: 46, carryOut: false)` *after* the `nbCode` block? (ie: so the sample is tested and the test is not shown in the docs). Does the CI fails if the doAssert is false?

If so, this would make sure the examples keep working and kept up to date.